libraries/radwordsprocessing/concepts/fields/sequence-field.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Sequence Field
+page_title: Sequence Field
+description: MergeField field is a Field element that contains a reference to a data field by its name.
+slug: radwordsprocessing-concepts-sequence-field
+tags: sequence,field
+published: True
+position: 8
+---
+
+# Sequence Field
+
+
+
+SequenceField is a [Field]({%slug radwordsprocessing-concepts-fields%}) element that sequally numbers chapters, tables, figures, and other items in the document. 
+If you add, delete, or move an item and its respective Sequence field, you can update remaining Seq fields in the document to reflect the new sequence.
+
+## Field Syntax
+This is how the syntax of a Sequence field looks like: 
```

Comment:
I think it's not a mistake to use the full name (Sequence) when you are talking about the field but when you explicitly say SeqField it should be the short way. (Here the using of the full name is OK)